lookout/core/analyzer.py

```diff
@@ -68,7 +68,7 @@ class Analyzer:
     model_type = None  # type: Type[AnalyzerModel]
     name = None  # type: str
 
-    def __init__(self, model: AnalyzerModel, url: str, config: Mapping[str, Any]):
+    def __init__(self, model: AnalyzerModel, url: str, config: Mapping[str, Mapping[str, Any]]):
```

Comment:
the type should stay `Mapping[str, Any]` even though in our case it is more structured. It is explicit in the API that this parameter is free-form.